src/screens/MemberDetail/MemberDetail.tsx

```
@@ -48,502 +58,114 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
-        state: userData?.user?.address?.state,
-        city: userData?.user?.address?.city,
-        country: userData?.user?.address?.countryCode,
-        pluginCreationAllowed: userData?.appUserProfile?.pluginCreationAllowed,
-        image: userData?.user?.image || '',
-      });
-    }
-  }, [userData, user]);
+    isMounted.current = true;
+    const superAdmin = getItem('SuperAdmin');
+    superAdmin
+      ? (document.title = t('title_superadmin'))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)